webapp/src/main/java/org/apache/atlas/web/rest/BusinessPolicyREST.java

```diff
@@ -112,4 +113,45 @@ public void unlinkBusinessPolicy(@PathParam("policyId") final String policyGuid,
             RequestContext.get().endMetricRecord(metric);
         }
     }
+
+
+    @POST
+    @Path("/move/{assetId}")
+    @Timed
+    public void moveBusinessPolicy(
+            @PathParam("assetId") String assetId,
+            final MoveBusinessPolicyRequest request
+    ) throws AtlasBaseException {
+        // Start performance metric recording
+        AtlasPerfMetrics.MetricRecorder metric = RequestContext.get().startMetricRecord("moveBusinessPolicy");
```

Comment:
This can be avoided, overall API metrics can be found in API audits loggings as well as the Perf loggings
AtlasPerfMetrics.MetricRecorder is primarily used for method-level metrics